build.py

```diff
@@ -193,6 +192,79 @@ def build_assess_authenticator_assets(remove_existing=False):
     static_assets.build_bundles(static_folder="static")
 
 
+def build_monolith_assets(static_dist_root="static/monolith", remove_existing=False) -> None:
+    MONOLITH_DIST_PATH = "./" + static_dist_root
+    MAIN_DIST_PATH = "./static"
+    GOVUK_URL = "https://github.com/alphagov/govuk-frontend/releases/download/v5.8.0/release-v5.8.0.zip"
```

Comment:
this is still 5.8.0 - I think if we're rolling back the python libs, we should keep using a compatible version of govuk frontend? is this too controversial?

Reply:
gaaaah I thought I removed that change all together!!